### ISSUE TITLE
hexa and 0

### ISSUE THREAD

**Issue by NotSqrt** [mpaland/printf#28]
Hi,

It seems that `printf("%#x", 0)` outputs `0` instead of `0x0`.

Is that normal ?

Thanks !

**Comment by mpaland** [comment]
Indeed, that's not a bug, it's a feature defined by the standard. I don't like it as well, but yeah, the specification has the last word 😉
